src/xturing/self_instruct/README.md

```diff
@@ -0,0 +1,17 @@
+# Dataset generation
+
+Process:
```

Comment:
@StochasticRomanAgeev should we delete this or improve this?

Reply:
We can add reference to .ipynb, because we have the same text there